CHANGELOG.md

```diff
@@ -1,4 +1,201 @@
-### [Versions](https://material-ui.com/versions/)
+### [Versions](https://material-ui.com/versions/
+
+## 5.0.0-alpha.38
+
+<!-- generated comparing v5.0.0-alpha.37..next -->
+
+_Jun 23, 2021_
+
+Big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 We have completed all the [breaking changes](https://github.com/mui-org/material-ui/issues/20012). The plan to release 5.0.0-beta.0 is on July 1st and will start to promote its usage over v4
+
```

Comment:
We could put an highlight on the design changes:

- New link style
- New slider style
- #26697 this one is not insignificant